library/std/src/lib.rs

```diff
@@ -282,6 +282,7 @@
 #![feature(int_log)]
 #![feature(into_future)]
 #![feature(intra_doc_pointers)]
+#![feature(io_safety)]
```

Comment:
```suggestion
#![cfg_attr(bootstrap, feature(io_safety))]
```

Reply:
I don't think the problem is it not being available during bootstrap, but instead it not being available on this specific target.

Reply:
Hay, that makes much more sense. Sorry for the noise.